device_backends/LogicalNameMapping/src/LNMDoubleBufferPlugin.cc

```diff
@@ -0,0 +1,124 @@
+#include "LNMDoubleBufferPlugin.h"
+
+#include "BackendFactory.h"
+
+std::string getValue(const std::map<std::string, std::string>& m, std::string key) {
```

Comment:
place in anonymous namespace to avoid conflict with a `getValue()` function declared elsewhere.

Reply:
alternatively, just use `std::map::at()` instead, and catch and rethrow the exception (only one catch+rethrow required for all calls)

Reply:
Code refactored. getValue is not needed anymore.  std::map::at() used to search value by key in map.